javascript/scanner.js

```diff
@@ -42,7 +42,9 @@ const scanForReflexesOnElement = (element, controller = null) => {
 
     const parentControllerElement = element.closest(`[data-controller~=${controllerName}]`)
 
-    if (!parentControllerElement) {
+    const elementPrevisoulyHadStimulusReflexController = (element === parentControllerElement && controllerName === 'stimulus-reflex')
```

Comment:
You have a typo here!

Reply:
Thanks, fixed in: https://github.com/stimulusreflex/stimulus_reflex/commit/6348953fb75299f696eab5dd4590e20699bf8c78